config/console/console_init.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: console
+spec:
+  selector:
+    matchLabels:
+      app: mcg-ms-console
+  template:
+    metadata:
+      labels:
+        app: mcg-ms-console
+    spec:
+      containers:
+        - name: mcg-ms-console
+          image: mcg-ms-console:latest
+          resources:
+            limits:
```

Comment:
can we add "request" also here?

Reply:
In odf/console, we dont have requests. Does it mean it runs with lower resources as well but limit is required to have boundary condition ?